basic/src/test/java/org/jboss/hal/testsuite/test/configuration/logging/RootLoggerTestCase.java

```diff
@@ -64,12 +65,18 @@ public void editFilterSpec() throws Exception {
     @Test
     public void editHandlers() throws Exception {
         String handler = "testHandler_" + RandomStringUtils.randomAlphanumeric(5);
+        ModelNodeResult defaultValue = operations.readAttribute(ROOT_LOGGER_ADDRESS, "handlers");
```

Comment:
In that case the cleanup logic has to be more complicated and harder to read IMO since it should either write or undefine attribute depending on the result of the opening read operation. Is it worth it?